### PR TITLE
LG-61 Fixes validation for front page feedback form

### PR DESCRIPTION
**Pull request by TaylorAkin**
LG-61 Fixes validation for front page feedback form

     - comments out code in scripts.js, which sercumvented the capthca validation
     - updated honeypot config
     - moved status messages block to top of lex_home block layout

### CODE REVIEW

**Review by ChrispyAPAX — APPROVED** [review]
looks good! nice work.

File: config/sync/block.block.fieldblock_page_section.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ dependencies:
   config:
     - field.storage.node.field_page_section
   module:
-    - ctools
```
Comment (chris-allen) [inline config/sync/block.block.fieldblock_page_section.yml:8]: Any thoughts on why this happened?

File: themes/custom/lex/js/scripts.js [hunk 2]
```diff
@@ -2,34 +2,36 @@
...
-      success: function(response) {
-        accordion.find('.js-success').removeClass('visually-hidden');
-        submitBtn.val(submitBtnOriginalVal);
-      },
-      error: function(response) {
-        accordion.find('.js-error').removeClass('visually-hidden');
-      }
-    });
-  });
+  // $('#js-feedback-button').find('a').click(function () {
```
Comment (chris-allen) [inline themes/custom/lex/js/scripts.js:5]: Need a comment with an explanation here. At first glance, this just looks like we're removing important behavior.

**Review by chris-allen — CHANGES_REQUESTED** [review]

File: config/sync/block.block.fieldblock_page_section.yml (hunk 1 above)
Comment (TaylorAkin) [reply config/sync/block.block.fieldblock_page_section.yml:8]: I don't know exactly why, for whatever reason it seems that it's a causation of adjusting the block layout for the Lexington theme.  There are multiple places here where ctools is being interchanged with node. After testing the site is working as expected. So I presume ctools isn't a needed dependency for these blocks.

**Review by chris-allen — APPROVED** [review]

**Pull request merged**